src/IW/Core/SqlArray.hs

```diff
@@ -7,5 +7,5 @@ import Database.PostgreSQL.Simple.Types (PGArray (..))
 
 newtype SqlArray a = SqlArray { unSqlArray :: [a] }
     deriving stock   (Generic, Show)
-    deriving newtype (Eq, ToJSON)
+    deriving newtype (Eq, Ord, ToJSON)
```

Comment:
I wonder why `Ord` instance is required? :thinking:

Reply:
Don't need. Result of copy and paste. Will fix this.